### DISCUSSION TITLE
YouTube Playlist and Watch Later Sync

### DISCUSSION THREAD

**Discussion by Drallas** [FreeTubeApp/FreeTube#5842] [General]
I love using Freetube for private, ad-free video watching, but I miss the ability to sync my “Watch Later” and subscribed channels / playlists with YouTube. Is there a way to sync these in a privacy-friendly way, maybe using an intermediate storage solution to keep data private?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
No there is unfortunately no way todo this in a privacy friendly way
